pages/api/rooms/create.js

```diff
@@ -30,43 +30,45 @@ export async function GenerateRoom() {
 
 
 export default async function link(req, res) {
-  await dbConnect()
-  const user = (await auth0.getSession(req)).user
-  const fellow = await Fellow.findOne({ username: user.nickname })
+  try {
+    await dbConnect()
+    const user = (await auth0.getSession(req)).user
+    const fellow = await Fellow.findOne({ username: user.nickname })
 
-  if (fellow.room) {
-    res.json({ message: fellow.room })
-  }
+    if (fellow.room) {
+      res.json({ message: fellow.room })
+    }
 
-  let value = await Fellow.aggregate([
-    {
-      $match: {
-        online: true,
-        username: { $ne: user.nickname }
-      }
-    },
-    {
-      $group: {
-        _id: "$podId",
-        members: { $push: "$_id" }
+    let value = await Fellow.aggregate([
+      {
+        $match: {
+          online: true,
+          username: { $ne: user.nickname }
+        }
+      },
+      {
+        $group: {
+          _id: "$podId",
+          members: { $push: "$_id" }
+        }
       }
+    ])
+    if (value.length == 0) {
+      res.json({
+        message: "Wait For other fellow to join"
+      })
+      Fellow.update({ username: user.nickname }, { $set: { online: true } })
+    } else {
+      const room = await GenerateRoom()
+      const members = [value[0].members[0], value[0].members[1], value[0].members[2]];
+      const redirect = await Redirect.create({ url: `https://videolink2me.com/o/${room.short_id}`, members })
```

Comment:
```suggestion
      const redirect = await Redirect.create({ url: `https://meet.jit.si/{uuid}`, members })
```
Where uuid is a generated uuid